### ISSUE TITLE
controlflow blocked on a chan cause  session.Player().LoadTrack() never return 

### ISSUE THREAD

**Issue by chfanghr** [librespot-org/librespot-golang#14]
line 73
![screenshot_20181224_170459](https://user-images.githubusercontent.com/40087668/50395373-77f0ae80-079e-11e9-97fc-7fd4f8d72d2e.png)

I found that no function send anything to the chan.
Here is my main.go code
```go
package main

import (
	"io/ioutil"
	"librespot"
	"librespot/utils"
)

const (
	username = "chfanghr@gmail.com"
	blob     = "blob.bin"
	trackid  = "0JiVRyTJcJnmlwCZ854K4p"
)

func main() {
	buf, err := ioutil.ReadFile(blob)
	if err != nil {
		panic(err)
	}
	session, err := librespot.LoginSaved(username, buf, "testclient")
	if err != nil {
		panic(err)
	}
	tr, err := session.Mercury().GetTrack(utils.Base62ToHex(trackid))
	if err != nil {
		panic(err)
	}
	tr.String()
	files := tr.GetFile()
	_ = len(files)

	audioFile, err := session.Player().LoadTrack(files[0], tr.GetGid())
	_ = audioFile
	ch := make(chan int)
	<-ch
}
```

**Comment by chfanghr** [comment]
I am so sure that this problem can not caused by my network .I tried both direct connection and proxy ,but both failed.And my official Spotify client work perfectly fine😂😂😂😂

**Comment by ViRb3** [comment]
Duplicate of #8 . Check #12 for potential fix

**Comment by chfanghr** [comment]
Thanks. Problem sloved.
